src/Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter.tsx

```diff
@@ -58,6 +73,16 @@ export const QuickMultipleSelectAlertFilter: FC<QuickMultipleSelectAlertFilterPr
           )
         })}
       </GridColumns>
+
+      {hasMore && (
+        <Clickable
+          mt={2}
+          onClick={() => setExpanded(visibility => !visibility)}
+          textDecoration={"underline"}
+        >
+          {isExpanded ? "Hide" : "Show more"}
+        </Clickable>
+      )}
```

Comment:
This toggle somewhat duplicates the functionality of Palette's `<ShowMore />` since that component did not actually work with this usage (it did very strange things to the size of the modal, and didn't yield to any quick fixes on my part).

But since it is baked into this `QuickMultipleSelectAlertFilter` component, any future open-ended lists that we add to this UI can benefit from this as well.

Reply:
cc @olerichter00 and @tam-kis

I just noticed that there was some back-and-forth about this a while ago in https://github.com/artsy/force/pull/13106 and https://github.com/artsy/force/pull/13156, where I believe you ran into some of these same^ layout problems.

The Story added in d80387378feffa4c217d20c2398487d8dc4e2bd0 demonstrates this new behavior:

![trunc out gif 5 128](https://github.com/artsy/force/assets/140521/e674e7a7-923b-41a6-9872-02978c8ebd6d)